example-config.json

```diff
@@ -1,135 +1,207 @@
-          "☕": "743142812257878027",
-          "🖲️": "731858559276679198",
-          "✨": "743142559043551363"
-        }
-      },
-      "754441222424363088": {
-        "770046936957845514": {
-          "🔪": "772945148937109546"
+        "reactionRoles": {
+            "898041838701781013": {
+                "918288000071381002": {
+                    "📣": "",
+                    "♻️": ""
+                },
+                "918240983362183211": {
+                    "🕹️": "",
+                    "🎥": "",
+                    "🎉": "",
+                    "❔": "",
+                    "⌛": ""
+                },
+                "918242254018199633": {
+                    "🐍": "",
+                    "☕": "",
+                    "🖲️": "",
+                    "✨": ""
+                }
+            },
+            "754441222424363088": {
+                "770046936957845514": {
+                    "🔪": ""
+                }
+            }
+        },
+        "Roles": {
+            "Owner": "",
+            "CoOwner": "",
+            "StaffManager": "",
+            "BotAdmin": "",
+            "Admin": "",
+            "Moderator": "",
+            "Helper": "",
+            "Staff": "",
+            "NewTicket": "",
+            "BetaTester": "",
+            "Booster": "",
+            "Donator": "",
+            "Client": "",
+            "Member": "",
+            "Bot": "",
+            "Muted": "",
+            "TicketBan": "",
+            "Announcements": "",
+            "Changelogs": "",
+            "Giveaways": "",
+            "Polls": "",
+            "GameNights": "",
+            "MovieNights": "",
+            "StaffUpdates": "",
+            "Others": ""
         }
-      }
-    }
-  },
-  "DB": {
-    "MongoDB": "mongodb - unused"
-  },
-  "Pterodactyl": {
-    "hosturl": "https://panel.danbot.host",
-    "apikey": "",
-    "apikeyclient": ""
-  },
-
-     "Email": {
-    "Host": "mail.danbot.host",
-    "Port": "587",
-    "User": "no-reply@danbot.host",
-    "Password": "",
-    "From": "no-reply@danbot.host"
-  },
-    
-  "Email2": {
-    "Host": "smtp.sendgrid.net",
-    "Port": "587",
-    "User": "apikey",
-    "Password": "",
-    "From": "noreply@danbot.host"
-  },
-
-  "Nodes": {
-      "PNode1": "",
-      "PNode2": "",
-      "PNode3": "",
-      "PNode4": "",
-      "Dono1": "",
-      "Dono2": "",
-      "Dono3": ""
-  },
-  "Servers": {
-      "KS-US-1": "",
-      "DE-EU-1": "",
-      "DE-EU-2": "",
-      "DE-EU-3": "",
-      "LO-UK-1": ""
-  },
-  "Tunnels": {
-      "US-1": "",
-      "UK-1": ""
-  },
-  "VPN": {
-      "US-1": "",
-      "UK-1": "",
-      "EU-1": "",
-      "AU-1": "",
-      "FR-1": ""
-  }
-}
+    },
+    "DB": {
+        "MongoDB": ""
+    },
+    "Pterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
+    },
+    "PrivPterodactyl": {
+        "hosturl": "",
+        "apikey": "",
+        "apikeyclient": ""
+    },
```

Comment:
Isn't this on an entirely different bot?

Reply:
Before stuff.